api/secrets/post.js

```diff
@@ -77,6 +77,9 @@ function _post(bag, next) {
   if (_.has(bag.reqBody, 'rootToken'))
     bag.config.rootToken = bag.reqBody.rootToken;
 
+  if (_.has(bag.reqBody, 'isShippableManaged'))
+    bag.config.isShippableManaged = bag.reqBody.isShippableManaged;
```

Comment:
what if the `isShippableManged` flag is not passed, are we setting this to a default value somewhere?